x-pack/plugins/apm/public/components/app/service_map/Popover/backend_contents.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { ContentsProps } from '.';
+import { NodeStats } from '../../../../../common/service_map';
+import { useUrlParams } from '../../../../context/url_params_context/use_url_params';
+import { useApmParams } from '../../../../hooks/use_apm_params';
+import { useApmRouter } from '../../../../hooks/use_apm_router';
+import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
+import { StatsList } from './stats_list';
+
+export function BackendContents({ nodeData }: ContentsProps) {
+  const { query } = useApmParams('/service-map');
```

Comment:
this breaks on the /service/:serviceName/service-map page. I don't really have a great solution for that right now other than using a wildcard (`/*`) or passing query to `BackendContents`

Reply:
Does `*/service-map/*` work?

Reply:
nah, it's not that clever (yet 😬 ), but you can do optional route matching with a second boolean argument, and then have two `useApmParams` calls

Reply:
^ does work

Reply:
Fixed in fa5fdbda0ed.